src/components/RefinementList/RefinementList.js

```diff
@@ -80,36 +81,36 @@ class RefinementList extends React.Component {
   //
   // Finally, we always stop propagation of the event to avoid multiple levels RefinementLists to fail: click
   // on child would click on parent also
-  handleItemClick(value, e) {
-    if (isSpecialClick(e)) {
+  handleItemClick({facetValueName, originalEvent, isRefined}) {
```

Comment:
Nope, not a public API, still found it easier than having to know argument positions.
What do you think?